include/private/config/locale/wrapper.h

```diff
@@ -29,6 +29,8 @@
 #	 include "private/config/locale/de-de.h"
 #  elif defined USE_LOCALE_SV_SE
 #    include "private/config/locale/sv-se.h"
+#  elif defined USE_LOCALE_SK_SK
+#   include "private/config/locale/sk-sk.h"
```

Comment:
slight indentation mismatch here; you just need one more space to bring it into line with everything else